packages/tools/pie-wrappers-react/scripts/react-gen-wrapper.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs-extra';
+import { appendFile } from 'fs'
+
+// fetches custom-elements.json file from the root
+const loadJSON = (path) => JSON.parse(fs.readFileSync(new URL(path, import.meta.url)));
```

Comment:
First time I see the usage of the URL constructor for reading local files. 

Are there any advantages over `path.join()` or `path.resolve()`?

In the PIE codebase I see the usage of `path.resolve` as well as `path.join`. I'd suggest sticking to a single approach for the whole repo, if possible.

I prefer `path.resolve` as it returns an absolute path. It makes debugging easier and doesn't have any downsides.